src/components/SendAsset/SendAsset.js

```diff
@@ -166,6 +166,8 @@ const SendAsset = ({
 
     setSubmitPressed(true);
 
+    selectedContact.ensName = await lookupAddress(selectedContact.ethAddress);
```

Comment:
@vignesha22 the function `lookupAddress` in `common.js` possibly returns null, what happens in this scenario?